### ISSUE TITLE
Accessibility And Loading Problems

### ISSUE THREAD

**Issue by jakescooty** [wagwan-piffting-blud/EAS-Tools#43]
I started testing the macro feature but i discovered 2 problems, firstly a visual accessibility problem as it is black before you hover on it which makes it hard to read and secondly the macros take forever to load and the loading bar glitches

**Comment by wagwan-piffting-blud** [comment]
>  it is black before you hover on it which makes it hard to read

Can you send me a good screenshot that shows me this issue please?

>  the macros take forever to load

This is totally normal and expected, since they are being emulated, I can't run at native speeds like with a computer running the true Audacity EXE. I tried to optimize it as best as I was able, but some macros will be slow to apply fully, some are near instant. All depends on the complexity (do they call existing built-in Audacity functions only? Do they call VST's, which need the emulation?) and how many steps need to run.

> and the loading bar glitches

In what way, can you elaborate on how it glitches?

**Comment by jakescooty** [comment]
@wagwan-piffting-blud 

<img width="1917" height="866" alt="Image" src="https://github.com/user-attachments/assets/963bebb8-9a12-444e-b272-fe319f2a04f0" />
the loading bar glitches by jumping too much back and forth and the percent is unreadable

**Comment by wagwan-piffting-blud** [comment]
Okay, I fixed both issues (macro list was hard-coded to dark mode in my haste to get it out, apologies), and the progress bar should now be much clearer when/if using a screen reader. Let me know if after the commit gets published you have any further comments or concerns in this issue, **I will keep the issue open until I can confirm everything is good on your end**.

**Comment by wagwan-piffting-blud** [comment]
@jakescooty The commit 13a4529 is now live, please give it a try and let me know your thoughts.

**Comment by jakescooty** [comment]
@wagwan-piffting-blud sorry for the late reply but yeah its better and i can read it thank you!
